### PR TITLE
Adding DG WholeBodyController and solo12 demo

### PR DESCRIPTION
**Pull request by jviereck**

### CODE REVIEW

File: python/mim_control/dg/wbc.py [hunk 1]
```diff
@@ -0,0 +1,144 @@
+"""This file is a demo for using the DG whole body controller.
```
Comment (jviereck) [inline python/mim_control/dg/wbc.py:1]: About where to place the file: I wanted to place it first under `mim_control/dynamic_graph/wbc.py`. However, that's the same path that is used by the dynamic graph plugins. Therefore, I went with `dg` for now.

Should we place this kind of code in the `dynamic_graph_py` subfolder maybe?

Comment (MaximilienNaveau) [reply python/mim_control/dg/wbc.py:1]: You should be able to put it in mim_control/dynamic_graph/wbc_graph.py

File: python/mim_control/dg/wbc.py [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+
+
+    def plug(self, robot, base_position, base_velocity):
+        # Args
+        #   robot; DGM robot device
+        #   base_position: The base position as a 7 dim vector signal
+        #   base_velocity: The base velocity as a 6 dim vector signal
+
+        # Create the input to the dg_robot.
+        base_pose_rpy = basePoseQuat2PoseRPY(base_position)
```
Comment (MaximilienNaveau) [inline python/mim_control/dg/wbc.py:120]: Where is this imported from?

File: python/mim_control/dg/wbc.py [hunk 3]
```diff
@@ -0,0 +1,144 @@
+"""This file is a demo for using the DG whole body controller.
+
+License BSD-3-Clause
+Copyright (c) 2021, New York University and Max Planck Gesellschaft.
+
+Author: Julian Viereck
+Date:   Feb 16, 2021
+"""
+
+from dg_tools.utils import *
```
Comment (MaximilienNaveau) [inline python/mim_control/dg/wbc.py:10]: Avoid the `import *`,  this is bad practice

File: python/mim_control/dg/wbc.py (hunk 2 above)
Comment (MaximilienNaveau) [reply python/mim_control/dg/wbc.py:120]: Ok from dg_tools, it is not intuitive with the `import *`

File: demos/demo_dg_whole_body_controller.py [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+
+import numpy as np
+np.set_printoptions(suppress=True, precision=3)
+
+from dg_tools.utils import *
+
+from robot_properties_solo.config import Solo12Config
+from dg_blmc_robots.solo.solo12_bullet import get_solo12_robot
+
+from mim_control.dg.wbc import WholeBodyController
```
Comment (MaximilienNaveau) [inline demos/demo_dg_whole_body_controller.py:18]: `from mim_control.dynamic_graph.wbc_graph import WholeBodyController`

**Review by MaximilienNaveau — CHANGES_REQUESTED** [review]

File: python/mim_control/dg/wbc.py (hunk 1 above)
Comment (jviereck) [reply python/mim_control/dg/wbc.py]: Good points, I moved the files under `mim_control/dynamic_graph/wbc_graph.py`

File: python/mim_control/dg/wbc.py (hunk 2 above)
Comment (jviereck) [reply python/mim_control/dg/wbc.py]: Thanks for pointing this out. I made the imports from `dg_tools` explicit.

**Comment by jviereck** [comment]
@MaximilienNaveau I've addressed your comments. Let me know if you want me to change anything else before merging.

**Review by MaximilienNaveau — APPROVED** [review]

**Pull request merged**